settings.json

```diff
@@ -1,753 +1,973 @@
+                },
+                {
+                    "name": "Bordered",
+                    "class": "templategrid-bordered"
+                },
+                {
+                    "name": "Lined",
+                    "class": "templategrid-lined"
+                },
+                {
+                    "name": "No styling",
+                    "class": "templategrid-stylingless"
+                }
+            ]
+        },
+        {
+            "name": "Hover style",
+            "type": "Toggle",
+            "description": "Enable template grid hover to make the rows hoverable.",
+            "class": "templategrid-hover"
+        }
+    ],
+    "GroupBox": [
+        {
+            "name": "Style",
+            "type": "Dropdown",
+            "description": "Choose one of the following styles to change the appearance of the groupbox.",
+            "options": [
+                {
+                    "name": "Brand Default",
+                    "oldNames": ["Default"],
+                    "class": "groupbox-default"
+                },
+                {
+                    "name": "Brand Primary",
+                    "oldNames": ["Primary"],
+                    "class": "groupbox-primary"
+                },
+                {
+                    "name": "Brand Success",
+                    "oldNames": ["Success"],
+                    "class": "groupbox-success"
+                },
+                {
+                    "name": "Brand Warning",
+                    "oldNames": ["Warning"],
+                    "class": "groupbox-warning"
+                },
+                {
+                    "name": "Brand Danger",
+                    "oldNames": ["Danger"],
+                    "class": "groupbox-danger"
+                },
+                {
+                    "name": "Transparent",
+                    "class": "groupbox-transparent"
+                }
+            ]
+        },
+        {
+            "name": "Callout style",
+            "type": "Toggle",
+            "description": "Enable the groupbox callout functionality to highlight the importance of the groupbox.",
+            "class": "groupbox-callout"
+        }
+    ],
+    "StaticImageViewer": [
+        {
+            "name": "Style",
+            "type": "Dropdown",
+            "description": "Choose the style of your image.",
+            "options": [
+                {
+                    "name": "Rounded",
+                    "class": "img-rounded"
+                },
+                {
+                    "name": "Thumbnail",
+                    "class": "img-thumbnail"
+                },
+                {
+                    "name": "Circle",
+                    "class": "img-circle"
+                }
+            ]
+        },
+        {
+            "name": "Center",
+            "type": "Toggle",
+            "description": "Align the image in the center of an element.",
+            "class": "img-center"
+        },
```

Comment:
Introduce new design properties for StaticImageViewer to help with fitting an image